### PR TITLE
feat:-variant of badges 

### PR DESCRIPTION
**Pull request by shubhamkumar648**

### CODE REVIEW

File: styles/badge.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+.badge__container {
+    position: relative;
+    width: fit-content;
+}
+
+
```
Comment (rahulcs754) [inline styles/badge.css]: don't leave space

File: styles/badge.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    color: var(--lightwhite-color);
+}
+
+.hert_icon {
+    right: -4px;
+    top: -7px;
+    background-color: var(--danger-color);
+    text-align: center;
+    color: var(--lightwhite-color);
+
```
Comment (rahulcs754) [inline styles/badge.css:46]: remove space

**Review by rahulcs754 — COMMENTED** [review]
after checked your code please provide description  and output of your code

File: styles/badge.css (hunk 1 above)
Comment (shubhamkumar648) [reply styles/badge.css]: okay

File: styles/badge.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+.cart_icon {
+    right: -3px;
+    top: -3px;
+    background-color: var(--info-color);
+    text-align: center;
+    color: var(--lightwhite-color);
+}
+
+.hert_icon {
```
Comment (tanishq20) [inline styles/badge.css]: typo: `heart_icon`

File: styles/badge.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    background-color: var(--success-color);
+}
+
+.status_dnd {
+    right: -3px;
+    bottom: -1px;
+    background-color: var(--dark-grey);
+}
+
+.cart_icon {
```
Comment (tanishq20) [inline styles/badge.css]: typo: `cart_icon`

**Review by tanishq20 — COMMENTED** [review]
suggested small change

**Review by rahulcs754 — APPROVED** [review]

**Review by tanishq20 — APPROVED** [review]

File: styles/badge.css (hunk 1 above)
Comment (shubhamkumar648) [reply styles/badge.css]: > okay
ok

**Pull request merged**